### ISSUE TITLE
IDDES model

### ISSUE THREAD

**Issue by jr144393** [gdtk-uq/gdtk#60]
I am interested in utilizing the IDDES model. However, upon its use, I receive the following error: 

Caught signal 8 (Floating point exception: floating-point divide by zero) 

Other turbulence models with the same case is running, but this one does not. Can someone assist me with the use of this model?

**Comment by uqngibbo** [comment]
Hi jr,

Could you try compiling the code in FLAVOUR=debug mode and posting the resulting output in this thread?

Also, what kind of case are you running?

Nick,

**Comment by jr144393** [comment]
Certainly. I will try this out today and post it here. The case is a simple flat plate, and it was just to test the IDDES for something similar to my case (and to see if I could invoke it properly).

**Comment by jr144393** [comment]
I will close this for now, time has prohibited investigating this further.
